app/less/app.less

```diff
@@ -10,12 +10,10 @@ html {
 body {
   color: #000;
   background: none;
-  font: 12px/18px 'Open Sans', "Lucida Grande", "Lucida Sans Unicode", Arial, Helvetica, Verdana, sans-serif;
+  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
```

Comment:
What about "Open Sans", "Lucida Grande", "Lucida Sans Unicode"?

Reply:
I've changed fonts to use "system font stack", and those fonts does not included.

I can add system-ui to font-list, but it will work only on webkit browsers and if you use it as default in your OS.

Reply:
@last-partizan, those font families that you took from snippets for Method 1 are used by Github and Wordpress plugin, though they don't include necessary families like Lucida Grande.

Such a decision might be made based on their user's browsers statistics.
The same link (![css-tricks article system font stack](https://css-tricks.com/snippets/css/system-font-stack/)) says that for Mavericks is used Lucida Grande.

And method #2 's description provides you with the stack that includes Lucida Grande.

Could you clarify why not include that one or just update this PR with Lucida Grande?

Reply:
@last-partizan, I'm sorry, now I realized that `-apple-system` solves that issue.

Reply:
From the Linux perspective, `Roboto,Oxygen-Sans,Ubuntu,Cantarell` makes no sense if you want to use the system font.

Reply:
What we should use for linux then?

Ubuntu - default in ubuntu, and looks good enough.
Cantarell - is default in gnome.